textattack/transformations/word_swap_change_location.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+
+from textattack.shared.data import NAMED_ENTITIES
+from textattack.transformations import Transformation
+
+
+def cluster_idx(idx_ls):
```

Comment:
Hi, ok, so this `cluster_idx` seems to work well, but luckily it's been solved for us already:

```
import more_itertools as mit
>>> data = [1, 2, 3, 5, 6, 7]
>>> [list(group) for group in mit.consecutive_groups(data)]
```

so I think you can replace calls to `cluster_idx` with that one-liner (`[list(group) for group in ...]`).